expertise/service/config/celery_config.py

```diff
@@ -0,0 +1,22 @@
+from kombu import Exchange, Queue
+from kombu.serialization import registry
+
+task_default_queue = 'default'
+task_queues = (
+    Queue('userpaper', routing_key='expertise.service.celery_tasks.run_userpaper'),
+    Queue('expertise', routing_key='expertise.service.celery_tasks.run_expertise')
+)
+# CELERY_IMPORTS = ('tasks')
+task_ignore_result = False
+# broker_url = 'redis://localhost:6379/0'
+broker_url = 'redis://localhost:6379/0'
```

Comment:
can you use a database number 10? it is the same the matching system is using.

Reply:
Sure that can be fixed